Fixing network message names

Claydo9 opened the pull request.

Claydo9 committed:
Update cfc_restart.lua

Claydo9 committed:
Jenkins Compatibility

sarahsturgeon commented:
ok to test

sarahsturgeon commented on lua/ulx/modules/sh/cfc_restart.lua:
```diff
@@ -12,18 +12,18 @@ if CLIENT then
-
-            draw.WordBox( 0, ScrW()-tw, 10, txt, "TargetID", {r = 0, g = 0, b = 0, a = 180}, {r = 255, g = 0, b = 0, a = 255} )
-        end )
+            local tw, th = surface.GetTextSize( txt ) -- Jenkins doesn't like th because it's unused but I have to leave it there because surface.GetTextSize() returns 2 things
```
```suggestion
            local tw, _ = surface.GetTextSize( txt )
```

I think technically if the second var is unused, you can do this:
```suggestion
            local tw = surface.GetTextSize( txt )
```

Claydo9 committed:
Update lua/ulx/modules/sh/cfc_restart.lua

Co-Authored-By: Brandon Sturgeon <sturgeonb4@gmail.com>

Claydo9 committed:
Update cfc_restart.lua

Claydo9 committed:
Jenkins Compatibility no 2

Claydo9 committed:
Improving Code

Adding whitespaces between operators, fixing whitespace trails.

Claydo9 committed:
Replaced variable "thyme" with "time"

Claydo9 committed:
Improving Code pt 2.

Removing whitespace trails.

Claydo9 committed:
Test fix for shadowing issues

Claydo9 committed:
More Shadowing fixes

Claydo9 committed:
Variable "time" shadow fix test

Claydo9 committed:
Final Shadowing fix

fixed "local time = Entity( 0 ):GetNWFloat( "CFC_SERVER_RESTART", -1 )"
and "local diff = math.max( 0, SysTime() - SysTime() + time )"

Claydo9 commented:
Ok, I've finally fixed this silly code! it's ready brandon.

Claydo9 commented:
Someone will probably have to re-write this though, the code isn't very pleasant to read and some of the methods that the original creator used are just kind of bad i.e "Entity( 0 ):SetNWFloat( "CFC_SERVER_RESTART", math.max( 0, diff ) )"

plally commented on lua/ulx/modules/sh/cfc_restart.lua:
```diff
@@ -12,41 +12,37 @@ if CLIENT then
-
-            draw.WordBox( 0, ScrW()-tw, 10, txt, "TargetID", {r = 0, g = 0, b = 0, a = 180}, {r = 255, g = 0, b = 0, a = 255} )
+            local tw, _ = surface.GetTextSize( txt )
+            draw.WordBox( 0, ScrW() - tw, 10, txt, "TargetID", { r = 0, g = 0, b = 0, a = 180 }, { r = 255, g = 0, b = 0, a = 255} ) -- Says there is an "Unnecessary Parenthesies" but I see none
```
you  should probably use github comments for comments like this instead of putting it in the code

plally commented on lua/ulx/modules/sh/cfc_restart.lua:
```diff
@@ -12,41 +12,37 @@ if CLIENT then
     net.Receive( "CFC_SERVER_RESTART", function()
-        local thyme= Entity( 0 ):GetNWFloat( "CFC_SERVER_RESTART", -1 )
-        SVRestartHud( thyme )
+        local time = Entity( 0 ):GetNWFloat( "CFC_SERVER_RESTART", -1 )
```
i think it was originally changed to `thyme` to avoid shadowing the  global `time` variable
maybe changing it to something indicating what it actually represents  would be better than changing it back to `time`

sarahsturgeon replied:
is there a global variable time? I know there's `os.time` but I can't find anything about `time` being something (though I thought so at first, too)

plally replied:
ye, actually i dont think there is a  global time  variable
but that was the justification for doing it, well a `reserved keyword` was 
https://github.com/CFC-Servers/cfc_restart/commit/ab031e42fd127d34df057137a74fbb984cf68b19
:thinking:

plally commented on lua/ulx/modules/sh/cfc_restart.lua:
```diff
@@ -12,41 +12,37 @@ if CLIENT then
-CFC_SERVER_RESTART = {yes = false, thyme= 30}
-
-function ulx.svrestart( calling_ply, thyme, stop )
+CFC_SERVER_RESTART = { ["yes"] = false, ["time"] = 30 }
```
why did you change the table syntax to this?
personally i prefer defining tables like `{ yes = false, time = 30 }`

sarahsturgeon replied:
we were diagnosing an issue with jenkins thinking there was a shadowed var

plally commented on lua/ulx/modules/sh/cfc_restart.lua:
```diff
@@ -62,23 +58,19 @@ function ulx.svrestart( calling_ply, thyme, stop )
-    local thyme= math.max( 0, tonumber( thyme ) )
-
-    CFC_SERVER_RESTART = {yes = true, thyme= ( SysTime()+tonumber( thyme ) )}
+    CFC_SERVER_RESTART = { ["yes"] = true, ["time"] = SysTime() + tonumber( time ) }
```
same question about the table definition
it also might be worth it to just put it on multiple lines
e.g.
```lua
CFC_SERVER_RESTART = { 
    yes = true,
    time = etc,
}
```

Claydo9 committed:
Temporarily changed "local time

Claydo9 committed:
Fixed Trailing Whitespaces

line 16
